app/services/induction/change_programme.rb

```diff
@@ -3,7 +3,13 @@
 class Induction::ChangeProgramme < BaseService
   def call
     ActiveRecord::Base.transaction do
-      current_induction_record.changing!(end_date)
+      Induction::ChangeInductionRecord.call(
+        induction_record: current_induction_record,
+        changes: {
+          induction_status: :changed,
+          end_date:,
+        },
+      )
 
       Induction::Enrol.call(participant_profile:,
                             induction_programme: new_induction_programme,
```

Comment:
This will duplicate the current IR and give the new record a changed status, so 2 IRs with changed status and then add a new IR during the Enrol which doesn't seem right to me.